mne/time_frequency/tfr.py

```diff
@@ -269,6 +272,11 @@ def cwt_morlet(X, sfreq, freqs, use_fft=True, n_cycles=7.0, zero_mean=False,
     decim : int
         Decimation factor applied after time-frequency decomposition.
         Defaults to 1.
+    time_range : tuple
+        The time range to which the time-frequency decomposition should be applied.
+        If time_range = (t0, t1), tfr will be calculated for times in [t0, t1 - 1].
+        Defaults to (0, None) corresponding to the whole time range, from the first 
+        to the last time indices.
```

Comment:
Clarify that the time freq is computed across the entire epochs, and just subselected later on (else one can think there'll be some border artefact)